tests/backends/test_database.py

```diff
@@ -13,15 +13,15 @@ def setUp(self):
     def test_database_queries(self):
         # Read and set to default value
         with self.assertNumQueries(5):
-            self.config.INT_VALUE
+            _ = self.config.INT_VALUE
```

Comment:
this artificial assigments look very noisy, let`t disable this lint. 
(and several more places with this)

Reply:
I find the rule useful.

Reply:
why? what is the sense of assigment?

Reply:
Refactored https://github.com/jazzband/django-constance/pull/562/commits/3d4652c2a140d2dcf58d2ab01a699e9e56343c60